core/src/main/web/outputdisplay/bko-plot/plotutils.js

```diff
@@ -515,7 +528,69 @@
           hiddenParent.addClass("ng-hide");
         }
         return height;
+      },
+
+      base64Fonts: {},
+
+      getFontToInject: function(font) {
+        var defer = bkUtils.newDeferred();
+        var src = '';
+        for (var url in font.urlformats) {
+          if (font.urlformats.hasOwnProperty(url)) {
+            var format = font.urlformats[url];
+            if (this.base64Fonts[url] == null) {
+              this.base64Fonts[url] = this.base64Encode(this.getFileSynchronously(url));
+            }
+            src += "url('data:application/font-" + format + ";charset=utf-8;base64," + this.base64Fonts[url] + "') format('" + format + "'), ";
+          }
+        }
+        src = src.replace(/,\s*$/, "");
+        return '@font-face' + " { " +
+          "font-family: '" + font.fontFamily + "';" +
+          "src: " + src + ";" +
+          "font-weight: " + font.fontWeight + ";" +
+          "font-style: " + font.fontStyle + ";" +
+          " }\n";
+        return defer.promise;
+      },
+
+      //http://stackoverflow.com/questions/7370943/retrieving-binary-file-content-using-javascript-base64-encode-it-and-reverse-de
+      base64Encode: function(str) {
```

Comment:
http://stackoverflow.com/questions/23223718/failed-to-execute-btoa-on-window-the-string-to-be-encoded-contains-characte